### ISSUE TITLE
Issues with "Apple Intelligence is downloading"

### ISSUE THREAD

**Issue by A70411** [leminlimez/Nugget#132]
Managed to enable Apple Intelligence on iOS 18.1 yesterday, now soon it says that "Apple Intelligence is downloading". This has been saying this message for nearly 3 hours now.

Is there a fix for this, or any method I could try?

**Comment by Ak-2077** [comment]
I waited for 12 hrs and it showing same , is there any method to skip this and get apple interlligent and writing tools

**Comment by noobmaster69-source** [comment]
> Just stay proof for a long time, the downloading text will disappear finally. iPhone 15 & iPhone 15 Plus will not broke the Face ID, is still working

nope, iphone 15 or 16 pro still has face ID issues. I've tried both, and Apple's intelligence feature is still stuck in a downloading state.
![2](https://github.com/user-attachments/assets/409b0415-77f5-4534-bed7-e446ae84f72c)
![1](https://github.com/user-attachments/assets/5a0173dc-1109-4e66-907d-d000279e1ad4)

**Comment by chuck00000** [comment]
> > Just stay proof for a long time, the downloading text will disappear finally. iPhone 15 & iPhone 15 Plus will not broke the Face ID, is still working
> 
> nope, iphone 15 or 16 pro still has face ID issues. I've tried both, and Apple's intelligence feature is still stuck in a downloading state. ![2](https://private-user-images.githubusercontent.com/187731691/384259583-409b0415-77f5-4534-bed7-e446ae84f72c.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEwNTg5MjcsIm5iZiI6MTczMTA1ODYyNywicGF0aCI6Ii8xODc3MzE2OTEvMzg0MjU5NTgzLTQwOWIwNDE1LTc3ZjUtNDUzNC1iZWQ3LWU0NDZhZTg0ZjcyYy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwOFQwOTM3MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNjQzNjIxY2M5NjI3ZmU1MWU2ZDRlNzdmZDJlNTRhZTE3MTcwOTdiMTcyMzI2OTM4ZTI1MmQ4YWEyM2NhMjI5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-eCsNYWv1gRd9bLBv25yHJk2K0pNvmbjQ5TQ1rFMYfg) ![1](https://private-user-images.githubusercontent.com/187731691/384259592-5a0173dc-1109-4e66-907d-d000279e1ad4.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEwNTg5MjcsIm5iZiI6MTczMTA1ODYyNywicGF0aCI6Ii8xODc3MzE2OTEvMzg0MjU5NTkyLTVhMDE3M2RjLTExMDktNGU2Ni05MDdkLWQwMDAyNzllMWFkNC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwOFQwOTM3MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZmEzYzZhNDRmNDkxZWYwOThmMTczNjI2MjMwYTUwZTYzNWViNGM4NjVlOWU0NDUyNGZlMDVmZjcwYjhkM2M0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.g92_bJ624fk0bHXOckS87zTpz7q97yqMQBuuJqruuMc)

Face ID still working on my iPhone 15, check the video.

https://github.com/user-attachments/assets/8b2a862b-dc98-416a-9f38-02059463d1c7

**Comment by Ak-2077** [comment]
Brother, I waited at least 12 to 15 hours. You want me to wait more so I’ll wait, but are you sure it will gonna work?On 8 Nov 2024, at 12:29 PM, noobmaster69-source ***@***.***> wrote:﻿

Just stay proof for a long time, the downloading text will disappear finally. iPhone 15 & iPhone 15 Plus will not broke the Face ID, is still working

nope, iphone 15 or 16 pro still has face ID issues. I've tried both, and Apple's intelligence feature is still stuck in a downloading state.
2.jpg (view on web)
1.jpg (view on web)

**Comment by chuck00000** [comment]
> Brother, I waited at least 12 to 15 hours. You want me to wait more so I’ll wait, but are you sure it will gonna work?On 8 Nov 2024, at 12:29 PM, noobmaster69-source ***@***.***> wrote:﻿ Just stay proof for a long time, the downloading text will disappear finally. iPhone 15 & iPhone 15 Plus will not broke the Face ID, is still working nope, iphone 15 or 16 pro still has face ID issues. I've tried both, and Apple's intelligence feature is still stuck in a downloading state. 2.jpg (view on web) 1.jpg (view on web) —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Such a long time, It won't work. Usually wait for 1~2 hours max until working after spoof.  Fastest waiting time usually within 10 mins.


I have another 12 Pro Max, After updating to 18.2 beta 2. First time I spoofed, The AI with chatGPT is working, but Face ID not working; Then I revert it to use Face ID; After that, I spoof my Device again, but this time  the Apple Intelligence is always stuck downloading, Although is download completely before. I tried spoof many times, still the same.


But another iPhone 15 No matter how many times spoof, AI is not stuck downloading.

That's strange.

**Comment by Ak-2077** [comment]
That’s your luck, brother. I have iPhone 14 and it’s stuck downloading yesterday before sleeping. I downloaded Apple intelligent. I spoof my device and again to use Face ID spoof my device and it shows Apple is downloading. It took 12 hours and still same so I decided to remove it.On 8 Nov 2024, at 6:00 PM, chuck00000 ***@***.***> wrote:﻿

Brother, I waited at least 12 to 15 hours. You want me to wait more so I’ll wait, but are you sure it will gonna work?On 8 Nov 2024, at 12:29 PM, noobmaster69-source @.> wrote:﻿ Just stay proof for a long time, the downloading text will disappear finally. iPhone 15 & iPhone 15 Plus will not broke the Face ID, is still working nope, iphone 15 or 16 pro still has face ID issues. I've tried both, and Apple's intelligence feature is still stuck in a downloading state. 2.jpg (view on web) 1.jpg (view on web) —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.>

Such a long time, It won't work. Usually wait for 1~2 hours max until working after spoof.  Fastest waiting time usually within 10 mins.
I have another 12 Pro Max, After updating to 18.2 beta 2. First time I spoofed, The AI with chatGPT is working, but Face ID not working; Then I revert it to use Face ID; After that, I spoof my Device again, but this time  the Apple Intelligence is always stuck downloading, Although is download completely before. I tried spoof many times, still the same.
But another iPhone 15 No matter how many times spoof, AI is not stuck downloading.
That's strange.

**Comment by kuanchichao** [comment]
stuck at downloading on i14 pro max
